Clean ps.m3u

tilfizyawn opened the pull request.

Commit:
Clean ps.m3u

Commit:
Add Musawah back

Commit:
revert

LaneSh4d0w commented on channels/ps.m3u:
```diff
@@ -29,13 +23,11 @@ http://htvpalsat.mada.ps:8888/PBC/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="palestiniantv.ps" tvg-name="palestiniantv" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",palestiniantv
+http://palestiniantv.origin.technostreaming.net:8081/palestiniantv_source/live/playlist.m3u8
+#EXTINF:-1 tvg-id="WatarTV.ps" tvg-name="Watar TV" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",Watar TV
+http://htvint.mada.ps:8889/orient/index.m3u8
```
Watar TV and Orient Radio + has got the same link.

+ I don't seem to have any image... is it a FM radio ?

LaneSh4d0w commented on channels/ps.m3u:
```diff
@@ -29,13 +23,11 @@ http://htvpalsat.mada.ps:8888/PBC/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="PALESTINIANTV.ps" tvg-name="PALESTINIAN TV" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",PALESTINIAN TV
+http://51.255.84.28:8081/palestiniantv_source/live/playlist.m3u8
+#EXTINF:-1 tvg-id="palestiniantv.ps" tvg-name="palestiniantv" tvg-country="PS" tvg-language="Arabic" tvg-logo="" group-title="",palestiniantv
+http://palestiniantv.origin.technostreaming.net:8081/palestiniantv_source/live/playlist.m3u8
```
Same channel than the one above ?

BellezaEmporium committed:
Merge branch 'master' into patch-14

Commit:
Update ps.m3u

Commit:
Remove dupllicate

Commit:
Rename to Watar TV

It's Watar TV according to https://7alaradio.ps/index.html but the m3u8 is currently down

Commit:
Add 7ala TV [Geo-Blocked]
